zhmccli/_cmd_imageprofile.py

```diff
@@ -1294,9 +1356,34 @@ def imageprofile_update(cmd_ctx, cpc, imageprofile, **options):
                  # Property: 'zaware-master-pw'
                  help='The master password for IBM zAware. '
                  'Empty string sets no master password.')
-# TODO: Support for 'zaware-network-info' property
-# TODO: Support for 'zaware-gateway-info' property
-# TODO: Support for 'zaware-dns-info' property
+@optgroup.option('--zaware-network-info', type=str, required=False,
```

Comment:
Code duplication ... couldn't these help text strings go into constants, instead of verbatim repeating them?

Reply:
That makes it more difficult, and between create and update the text is btw not the same